utils/python/make_nexus_output_pretty.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+__author__ = 'Barry Baker'
+__email__ = 'barry.baker@noaa.gov'
+__license__ = 'GPL'
+
+import netCDF4 as nc
+import numpy as np
+from netCDF4 import date2num,num2date
+import datetime as dt
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter 
+
+def get_hemco_dates(time_file='HEMCO_sa_Time.rc'):
+    fi = open(time_file)
+    lines = fi.readlines()
+    fi.close()
+    for L in lines:
+        if L.startswith('START'):
+            start_base = dt.datetime.strptime(L.split()[1],'%Y-%m-%d')
+            start = dt.datetime.strptime(L, 'START:   %Y-%m-%d %H:00:00\n')
+        if L.startswith('END'):
+            end = dt.datetime.strptime(L, 'END:     %Y-%m-%d %H:00:00\n')
+        if L.startswith('TS_EMIS'):
+            ts_emis = float(L.split()[1].strip('\n'))
+    dates_offset = (start - start_base).total_seconds() / 3600
+    dates = [start + dt.timedelta(hours=x) + dt.timedelta(hours=dates_offset) for x in range(int((end - start).total_seconds() / 3600) + 1 )]
+    return dates,start_base
+
+def create_ds_dict(dset):
+    d = {}
+    for var in dset.variables.values():
+        d[var.name] = dset[var.name][:]
+    return d
+
+
+if __name__ == '__main__':
+
+    parser = ArgumentParser(description='Make the NEXUS output pretty', formatter_class=ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-s', '--src', help='input nexus file', type=str, required=True)
+    parser.add_argument('-g', '--grid', help='input grid file', required=False)
+    parser.add_argument('-t', '--read_hemco_time', help='Read HEMCO time file', default=True, required=False)
```

Comment:
```suggestion
    parser.add_argument('-t', '--read-hemco-time', help='Read HEMCO time file', default=True, required=False)
```
(argparse will convert when reading)

Reply:
Also the name makes it sound like a boolean flag, but it looks like it represents the file path here

Reply:
Also looks to be required, based on below